test/fixtures/func.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
     'calc(1px + -2vw - 4px)',
     'calc(((768px - 100vw) / 2) - 15px)',
     'bar(baz(black, 10%), 10%)',
-    '-webkit-linear-gradient(0)'
+    '-webkit-linear-gradient(0)',
+    'var(--foo)'
```

Comment:
@shellscape 
The tests for `var()` and all color functions above would ideally also check that the related meta property of `isVar` or `isColor` is set

Reply:
never mind, it's all visible within the snapshots